Changes in documentation (#1084)
SumoLogic/sumologic-kubernetes-collection: vagrant/README.md

vagrant ssh
```

> NOTICE: The directory with sumo-kubernetes-collection repository is synced with the virtual machine.

Reviewer: ```suggestion
NOTICE: The directory with sumo-kubernetes-collection repository on the host is synced with `/sumologic/` directory on the virtual machine.
```

Reviewer: ping @kkujawa-sumo

vagrant ssh
```

NOTICE: The directory with sumo-kubernetes-collection repository is synced with the virtual machine.

## Build

On the virtual machine, to perform docker image build and to run tests please use `build` target:

```bash
sumo-make build

> ```

Reviewer: keep the old one as alternative as well  `... or ...` (2 snippets)

## Collector

To install or upgrade collector please type:

```bash
sumo-make upgrade

> ```

Reviewer: ditto

To remove collector please use:

```bash
sumo-make clean

> ```

Reviewer: ditto

List of other useful options for `sumo-make`:

- `expose-prometheus` - exposes prometheus on port 9090 of virtual machine
- `expose-grafana` - exposes grafana on port 8080 of virtual machine
- `apply-avalanche` - run one pod deployment of avalanche (metrics generator)

## Test

> To quickly test whether sumo-kubernetes-collection works receiver-mock can be used.

Reviewer: ```suggestion
In order to quickly test whether sumo-kubernetes-collection works, one can use `receiver-mock` for that purpose.
```

🤷🏻

To check receiver-mock logs please use:
```
kubectl logs $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -n receiver-mock
```
To check metrics exposed by receiver-mock please use:
```
kubectl exec $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -it -n receiver-mock -- curl http://localhost:3000/metrics

> ```

Reviewer: Empty lines around snippet backticks

To remove collector please use:

```bash
sumo-make clean
```

or

```bash
/sumologic/vagrant/Makefile clean
```

List of other useful targets:

- `expose-prometheus` - exposes prometheus on port 9090 of virtual machine
- `expose-grafana` - exposes grafana on port 8080 of virtual machine
- `apply-avalanche` - run one pod deployment of avalanche (metrics generator)

## Test

In order to quickly test whether sumo-kubernetes-collection works, one can use `receiver-mock` for that purpose.

To check receiver-mock logs please use:

```
kubectl logs $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -n receiver-mock
```

To check metrics exposed by receiver-mock please use:

```
kubectl exec $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -it -n receiver-mock -- curl http://localhost:3000/metrics

> ```

Reviewer 2: We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

Author: > We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

I'll add this in a new PR

To remove collector please use:

```bash

> sumo-make clean
> 

Reviewer: ```suggestion
sumo-make clean
```

Reviewer: redundant empty line
